pipeline/test/transformers/addressables.test.ts

```diff
@@ -24,6 +24,25 @@ describe('addressables transformer', () => {
     }
   );
 
+  forEachPrismicSnapshot<ArticlePrismicDocument>(['articles'], isAddressable)(
```

Comment:
We should probably mock the fetchAndTransformWorks function so we aren't making real calls to the catalogue API for the test

Reply:
I can do it in a separate PR